README.md

```diff
@@ -3,34 +3,24 @@
 ## Usage
 
 ```
-mkdir ~/.s3cfg
-
-docker run -it --rm         \
-  --volume ~/.s3cfg:/.s3cfg \
-  --user $(id -u)           \
```

Comment:
Running in --user mode is to help prevent bugs in docker from allowing a root escape.  Probably not a huge deal but why remove it?

Reply:
Ah, I didn't know what that did so I pulled it out 😬 Probably sensible to add it back in...